addons/proton_scatter/src/scatter.gd

```diff
@@ -20,6 +20,10 @@ const ProtonScatterUtil := preload('./common/scatter_util.gd')
 @export_category("ProtonScatter")
 
 @export_group("General")
+@export var enabled := true:
+	set(val):
+		enabled = val
+		rebuild()
```

Comment:
Shouldn't this be guarded like:
```
if _is_ready:
  rebuild()
```
Otherwise, I assume it will be called on object creation (same issue we had with the chunks parameter)

Reply:
For some reason it does not seem to run on my end when I open the scene. 
However, I think it is a good idea anyway. I'll add it.